.cfignore

```diff
@@ -12,3 +12,5 @@ hses.zip
 temp/
 tests/
 .tmp/
+*.test.js
+!*.sql
```

Comment:
does this do it? Doesn't it need to be in the deploy config?

Reply:
according to chat gpt this should do it

Reply:
oh boy

Reply:
let's deploy to sandbox?

Reply:
works: https://tta-smarthub-sandbox.app.cloud.gov/api/admin/ssdi/list-queries